DataFormats/Math/interface/deltaPhi.h

```diff
@@ -38,7 +38,12 @@ namespace reco {
 
   template <typename T>
   constexpr T deltaPhi(T phi1, T phi2) {
-    return reduceRange(phi1 - phi2);
+    constexpr T epsilon = 1.e-13;
```

Comment:
why ```1.e-13``` ? shouldn't this be a multiple of ```std::numeric_limits<T>::epsilon()``` ?

Reply:
I was just copying from angle0to2pi. This epsilon() sounds better.